contents/docs/llm-analytics/_snippets/trace-event.mdx

```diff
@@ -0,0 +1,47 @@
+A trace is a group that contains multiple spans, generations, and embeddings. Traces can be manually sent as events or appear as pseudo-events automatically created from child events.
+
+Event Name: `$ai_trace`  
+
+| Property | Description |
+|----------|-------------|
+| `$ai_trace_id` | The trace ID (a UUID to group related AI events together)<br/>Must contain only letters, numbers, and special characters: `-`, `_`, `~`, `.`, `@`, `(`, `)`, `!`, `'`, `:`, <code>\|</code><br/>Example: `d9222e05-8708-41b8-98ea-d4a21849e761` |
+| `$ai_input_state` | The input of the whole trace<br/>Example: `[{"role": "user", "content": "What's the weather in SF?"}]` or any JSON-serializable state |
```

Comment:
```suggestion
| `$ai_input_state` | The input of the whole trace<br/>**Example: ** <br/>`[{"role": "user", "content": "What's the weather in SF?"}]` or any JSON-serializable state |
```

Maybe like the optional thing, a good bold next line between **example** and the examples might look better?